wurst/math/Matrices.wurst

```diff
@@ -0,0 +1,310 @@
+package Matrices
+import NoWurst
+import Vectors
+import String
+
+
+/* Great thanks to euclideanspace.com for wide explanations. */
+
+
+public let ZERO22 = mat2(0, 0, 0, 0)
+public let ZERO33 = mat3(0, 0, 0, 0, 0, 0, 0, 0, 0)
+
+public let IDENTITY22 = mat2(
+    1, 0,
+    0, 1)
+
+public let IDENTITY33 = mat3(
+    1, 0, 0,
+    0, 1, 0,
+    0, 0, 1)
+
+
+/* =========== 2X2 MATRIX =========== */
+/** Matrxi 2x2
+00 01
+10 11
+*/
+public tuple mat2(real rc00, real rc01, real rc10, real rc11)
+
+                     
+public function mat2.op_plus(mat2 m) returns mat2
+    return mat2(
+        this.rc00 + m.rc00, this.rc01 + m.rc01,
+        this.rc10 + m.rc10, this.rc11 + m.rc11)
+
+public function mat2.op_minus(mat2 m) returns mat2
+    return mat2(
+        this.rc00 - m.rc00, this.rc01 - m.rc01,
+        this.rc10 - m.rc10, this.rc11 - m.rc11)
+
+public function mat2.op_minus(real scalar) returns mat2
+    return mat2(
+        this.rc00 - scalar, this.rc01 - scalar,
+        this.rc10 - scalar, this.rc11 - scalar)
+
+public function mat2.op_plus(real scalar) returns mat2
+    return mat2(
+        this.rc00 + scalar, this.rc01 + scalar,
+        this.rc10 + scalar, this.rc11 + scalar)
+    
+public function mat2.op_mult(real scalar) returns mat2
+    return mat2(
+        this.rc00*scalar, this.rc01*scalar,

```

Comment:
You leave spaces around `-` and `+` but not `*`. Pls fix for `*`

Reply:
As far as I know an operation with high priority doesn't require space around it. In most languages. That means I should leave one space around ``-`` and ``+`` but shouldn't do that around ``*`` and ``/``. In this case you ask me to fix it for ``*`` but not for ``/``. Is there any wurst-style doc I need to learn?